### ISSUE TITLE
Investigate a system to opt in to ignoring axe violations

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#5874]
We could support something like data-a11y-ignore with axe rule ids to help reduce false positives.
